src/components/mods/FSPEventNames.ts

```diff
@@ -0,0 +1,46 @@
+import { EventNames } from "modattachr/lib/EventNames";
```

Comment:
Looks like your branch is out of date. You'll have to pull from master. This file shouldn't be here.